.github/actions/setup/action.yaml

```diff
@@ -0,0 +1,34 @@
+name: "Setup"
+description: "Downloads and loads build image"
+
+inputs:
+  image_name:
+    description: “Build image name”
```

Comment:
The double quotes here seems non-ascii chars, or at least different from the one in line 2?